README.md

```diff
@@ -31,25 +82,16 @@
 
 ## How it works
 
-- Uses Playwright to interact with the site
-- At the time of a test run, the framework spins up an Anvil fork of the desired network using Foundry
-- It then injects a wallet to the browser that uses the Anvil fork as its RPC 
-- It then states its chain ID to be that of the mainnet of the desired network and reports any “eth_sendTransaction” requests
+- Spins up an Anvil fork of the desired network at the time of test run
+- Uses Playwright to run browser interactions and engage with the site
+- Injects a wallet to the browser that's connected to the Anvil fork
+- Checks eth_sendTransaction and eth_sendRawTransaction RPC requests when instructed to via gui.validateContractInteraction
 
 
 ## Documentation
 
-For full documentation and examples, visit [guardianui.com](INSERT LINK).
-
-## Installation
-
-Install guardianui and its dependencies.
-
-```bash
-npm install @guardianui/guardianui
-```
-
-## Quick Start
+For full documentation and examples, visit [guardianui.com](https://docs.guardianui.com/).
```

Comment:
Since this link doesn't point to "guardianui.com" we should rename to be like "visit the [GuardianTest Docs](https://docs.guardianui.com/)."